src/types/classes/invoice.ts

```
@@ -0,0 +1,384 @@
+	/**
+	 * @description The constructor for the Invoice object
+	 * @param apiClient The API client used to make requests to the API
+	 * @param invoiceCreationData The data used to create the invoice
+	 */
+	constructor(apiClient: ApiClient, invoiceCreationData: InvoiceCreationData) {
+		// Set the API Client
+		this.apiClient = apiClient;
+
+		// Set the Creation Data
+		this.Line = invoiceCreationData?.Line ?? new Array();
+		this.CustomerRef = invoiceCreationData?.CustomerRef ?? null!;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)